test/integration/gecko_t_osx_1015_r8/serverspec/gw_checker_spec.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'spec_helper'
+
+# This test suite checks for the presence of the generic worker checker script
+describe file('/usr/local/bin/gw_checker.sh') do
+  it { should exist }
+  it { should be_executable.by('owner') }
+  it { should be_executable.by('group') }
+  it { should be_executable.by('others') }
+end
+
+# Experimental
```

Comment:
Any ideas why this part is failing @aerickson 

CircleCI 'sees' the script but doesn't 'see' the crontab [entry](https://github.com/mozilla-platform-ops/ronin_puppet/blob/86fbcf92ec85ba0cc6e7782c85abbac039c8eddf/modules/macos_gw_checker/manifests/init.pp#L17)